content/api/suppression-list.apib

```diff
@@ -4,48 +4,84 @@ description: Manage your suppression list - a list of recipient email addresses
+    + non_transactional (boolean) - Whether the recipient requested to not receive any non-transactional messages. Deprecated in favor of `type`
+    + subaccount_id (number) - Which subaccount the recipient is suppressed for, only returned if suppressed for a specific subaccount.
 
-### Insert or Update List Entries [PUT]
++ Sample
+    ```
+    {
+        "recipient": "recip@example.com",
+        "type": "non_transactional",
+        "source": "List Unsubscribe",
+        "description": "Unsubscribed using list unsubscribe header",
+        "created" : "2017-10-01T12:00:00+00:00",
+        "updated" : "2017-10-01T12:00:00+00:00",
+        "transactional": false,
+        "non_transactional" true
+    }
+    ```
 
-Bulk insert or update entries in the suppression list by providing a JSON object, with a `recipients` key containing an array of recipients to insert or update, as the PUT request body. Maximum size of the JSON object is 50mb. Maximum recipients at a time is 10,000. At a minimum, each recipient must have a valid email address and a suppression type: `transactional` or `non_transactional`. The optional `description` key can be used to include an explanation of what type of message should be suppressed.
+## Insert & Update [PUT /suppression-list/]
 
-If the recipient was added to the list by Compliance, it cannot be updated.
+Bulk insert or update entries in the suppression list.
 
-Please note that in the unlikely scenario where your receive a HTTP 5xx level error response while bulk loading, that some of your suppression entries may have been successfully inserted or updated. If this occurs, please re-submit your original request again for processing.
+If a recipient was added to the list by compliance, it cannot be updated.
 
-<banner status="warning"><strong>Deprecation Notice</strong>: The <tt>email</tt> attribute is deprecated. Use <tt>recipient</tt> instead.</banner>
++ Data Structure: PUT Request Body
+    + recipients: (array, required) - Array of recipient objects to add to or updates in the suppression list. Max size: 50mb. Max length: 10,000.
 
-<banner status="warning"><strong>Deprecation Notice</strong>: The attributes <tt>transactional</tt> and <tt>non_transactional</tt> are deprecated. Use <tt>type</tt> instead.</banner>
 
++ Data Structure: Recipient Object
+    + recipient (array, required) - Email address to be suppressed
+    + type (enum, required) - Type of suppression record
+        + transactional
+        + non_transactional
+    + description (string) - Explanation for the suppression
+
+Please note that in the unlikely scenario where your receive a HTTP 5xx level error response while bulk loading, only some of your suppression entries may have been successfully inserted or updated.
+If this occurs, please re-submit your original request again for processing.
 
 + Request (application/json)
```

Comment:
Order the responses so that the `200` is first. And maybe get rid of one of the `400`s.